ee/localserver/request-query.go

```diff
@@ -0,0 +1,33 @@
+package localserver
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+func (ls *localServer) requestQueryHandler() http.Handler {
+	return http.HandlerFunc(ls.requestQueryHanlderFunc)
+}
+
+func (ls *localServer) requestQueryHanlderFunc(res http.ResponseWriter, req *http.Request) {
+	query := req.URL.Query().Get("query")
```

Comment:
If you don't use POST, I think we should assume some encoding. base64 is probably fine. (Maybe need base64 for post too)

Reply:
we call the standard url encoding when we make the request from krypto-ec-middleware.go do you think that is not sufficient? I tested with curl like.

```
curl -G localhost:40978/query --data-urlencode "query=select * from kolide_launcher_info where branch in ('unknown') AND branch like '%unknown%'"
```

I can also add some complex sql statements to the tests to see how it handles `string -> json -> bytes ->  json -> string`

Reply:
lol, the code snippet I posted is actually doing a post 😕

Reply:
I think we should base64 encode it.

Reply:
ecmiddleware now b64 encoding url parameters